modules/imgcodecs/include/opencv2/imgcodecs.hpp

```diff
@@ -260,10 +260,20 @@ It provides support for looping, background color settings, frame timing, and fr
 struct CV_EXPORTS_W_SIMPLE Animation
 {
     //! Number of times the animation should loop. 0 means infinite looping.
+    /*! @note (GIF) When N is set, whether it is displayed N or N+1 times depends on the implementation of the user application.
+              Animated GIF with loop is extended with the Netscape Application Block(NAB), which it not a part of GIF89a specification.
+              See https://en.wikipedia.org/wiki/GIF#Animated_GIF .
+              This loop times behaviour has not been documented clearly.
```

Comment:
For more context: https://issues.chromium.org/issues/40459899 . WebP had the same problem where 1 loop coould not happen: https://groups.google.com/a/webmproject.org/g/webp-discuss/c/rJ2a0j1BNQI/m/XXqK918ZEQAJ